cost-analyzer/values.yaml

```diff
@@ -935,15 +935,6 @@ grafana:
   rbac:
     # Manage the Grafana Pod Security Policy
     pspEnabled: false
-  datasources:
```

Comment:
@jessegoodier Could you explain the reasoning behind removing `.Values.datasources`? I notice it's used here as well:

https://github.com/kubecost/cost-analyzer-helm-chart/blob/3a283b62bb914daf45bc4ead079eddbe0612e212/cost-analyzer/charts/grafana/templates/configmap.yaml#L24-L29

If we do decide to get rid of `datasources.yaml` we should also remove the above code snippet and test?

Reply:
I was struggling with the logic.

Can you take a look at the configmap logic now?